README.md

```diff
@@ -1,21 +1,77 @@
-# Monitorizare Vot - Platforma pentru votanti
+# Monitorizare Vot - Web platform for elections observers
```

Comment:
This one is not for the observers - it's for the general public, the voters, the citizens :)